examples/getstarted/src/index.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
    *
    * This gives you an opportunity to extend code.
    */
-  register({ strapi }) {},
+  register(app) {},
```

Comment:
Why are we no longer destructuring strapi here? I'm not sure we should have a diff at all in this file

Reply:
thanks for highlighting that, i was using the github commit system so wasn't checking well enough!